src/exec.sh

```diff
@@ -107,7 +105,10 @@ cd build-${TIMESTAMP} # IN BUILD DIRECTORY =============================
 tar -xzf ../uploads/$TARBALL
 testIt
 tar -cf ../screenshots/${TIMESTAMP}.tar "$TESTS_FOLDER/screenshots"
-cd - # IN ROOT DIRECTORY ==================================
+
+# CD TO ROOT DIRECTORY ==================================
+cd $DIR/..
+
```

Comment:
Did `testIt` change to make more than one `cd` call? I'm fairy certain this was working before...